cmd/dummy/main.go

```diff
@@ -17,19 +18,24 @@ func main() {
 	persistencePtr := flag.String("persist", "", "directory to use for a database")
 	flag.Parse()
 
+	logger := log.NewTmLogger(os.Stdout)
+
 	// Create the application - in memory or persisted to disk
 	var app types.Application
 	if *persistencePtr == "" {
 		app = dummy.NewDummyApplication()
 	} else {
 		app = dummy.NewPersistentDummyApplication(*persistencePtr)
+		app.(*dummy.PersistentDummyApplication).SetLogger(log.With(logger, "module", "dummy"))
 	}
 
 	// Start the listener
 	srv, err := server.NewServer(*addrPtr, *abciPtr, app)
 	if err != nil {
-		log.Fatal(err.Error())
+		logger.Error(err.Error())
+		os.Exit(1)
```

Comment:
What do you think? Do we need a helper method:

```
func Fatal(logger Logger, msg string, keyvals ...interface{}) {
  logger.Error(msg, keyvals...)
  os.Exit(1)
}
```

My opinion is that one line does not worth it.